scripts/release.sh

```diff
@@ -261,6 +261,10 @@ parse_common_arg() {
       STATIC_LINKING="false"
       shift
       ;;
+    --no-debug-symbols)
```

Comment:
suggestion: strip symbols is a well know thing

```suggestion
    --strip)
```

Reply:
strip isn't very clear on what this will eventually do?
`-C strip=debuginfo -C debuginfo=0` would avoid generating debug symbols as well. Also it wouldn't remove the entire symbols table, just the debug symbols.

Anyhow, this PR's new direction is to allow random build flags, so this might not matter all that much.

Reply:
On a related not - Have you considered splitting the debuginfo and symbols to a separate file using `split-debuginfo` , to reduce the size while still keeping potentially useful info around?

Reply:
This is no longer relevant, I think.

Reply:
Going with a generic RUSTFLAGS input to the release.sh script. This opens it up for custom build parameters.